app/src/main/java/no/usn/mob3000/domain/model/content/FaqStruct.kt

```diff
@@ -0,0 +1,30 @@
+/**
+ * These are faq related data structures needed for the UI layer.
+ *
+ * TODO: Abstraction, especially for "UpateData"
+ *
+ * @author 258030
+ * @created 2024-10-30
+ */
+
+package no.usn.mob3000.domain.model.content
+
+import kotlinx.datetime.Instant
+
+data class FAQData(
+    val faqId: String,
+    val createdAt: Instant,
+    val modifiedAt: Instant,
+    val createdByUser: String,
+    val title: String,
+    val summary: String,
+    val content: String,
+    val isPublished: Boolean
+)
+
+data class FaqUpdateData(
```

Comment:
Working with data structures that don't represent the database tables, shouldn't make it necessary to have different structs for "updated" data. An object should always represent its latest state, and so, the updated data should just be written to the `FAQData` struct instead.